solidity/app/cl_wallet.js

```diff
@@ -0,0 +1,24 @@
+require('./cl_utils.js')
+
+const Tx = require('ethereumjs-tx')
+const Wallet = require('ethereumjs-wallet')
+
+global.clWallet = global.clWallet || {
```

Comment:
Can this use `module.exports` instead of the `global`?

Reply:
Uses a combination of the two now. State changes are saved globally, but the object can be imported as a local variable through require/module.exports.